packages/tests/src/middleware/superuserCheckSigner.ts

```diff
@@ -159,6 +208,37 @@ describe("Middleware: superuserCheckSigner", () => {
     );
   });
 
+  it("cannot execute with an unauthorized signer if the transaction was proposed by none (middleware error)", async () => {
+    // propose the Cryptid transaction
+    const { proposeTransaction, transactionAccount, proposeSigners } =
+      await authorizedCryptid.propose(makeTransaction());
+    // this will be signed by the non-did signer
```

Comment:
Actually noticed that this comment is wrong (copy-paste) in this example it's specifically a DID signer.